k8s/aat/common/rd/judicial-api.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: rd-judicial-api
+  namespace: rd
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: rd-judicial-api
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: rd-judicial-api
+    version: 0.0.2
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/rd/judicial-api:prod-e576d62a
```

Comment:
This is an image that exists? If it doesn't just put latest and flux will pickup the latest one, if it doesn't exist flux will fail to apply the change